build.gradle

```diff
@@ -15,7 +15,7 @@ buildscript {
         classpath 'org.jetbrains.dokka:dokka-gradle-plugin:0.9.13'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.6'
         classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.1.0'
-        classpath 'org.junit.platform:junit-platform-gradle-plugin:1.0.0-M4'
+        classpath 'org.junit.platform:junit-platform-gradle-plugin:1.0.0-M6'
```

Comment:
This is kind of breaking, since IJ supports only M5 (I guess?) in the most recent EAP. There is [a version matrix](http://junit.org/junit5/docs/current/user-guide/#running-tests-ide-intellij-idea), but not sure how it is keeping relevant. M5 actually marked failed tests and successful, so I’m hesitant.